build.gradle

```diff
@@ -247,6 +247,7 @@ task build {
 }
 
 task codeCoverageReport(type: JacocoReport) {
+    dependsOn('build')
```

Comment:
Shouldn't we update the other task dependencies as well?

i.e: `'http-ballerina-tests:codeCoverageReportBal'` should depends on the `http-ballerina:build` right?

Reply:
yes, that's my fault, need to update `http-ballerina-tests:codeCoverageReportBal`dependency. I think i can remove the build dependency from here. am I right?

Reply:
Yes, if we add dependencies for the other two tasks, we can remove the dependency here.